testapps/StrongNamedApp/StrongNamedApp.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk.Web" ToolsVersion="15.0">
```

Comment:
Nit: this and other testapps files can be cleaned up. Remove redundant attributes (AssemblyName), use one-line syntax for PackageReference, remove the unnecessary 'DefineConstants RELEASE' block at the end.